test/testdata/lsp/fast_path/static_field_change_type__use.rb

```diff
@@ -1,7 +1,11 @@
 # typed: strict
+# spacer for exclude from update
```

Comment:
When using the `exclude-from-file-update: true` assertion, the contents of this file will never be edited, so it has to contain exactly what we want the file to contain for all generations of the test.

But since subsequent files will "insert" the `exclude-from-file-update: true` assertion, we need a spacer line here so that `error` assertions (which are read out of the file by the test runner, not processed by the LSP server) are on the expected lines.

If we did not have this spacer, it would mean that for example our `error:` assertions in `.1.rbupdate` would be off by one line too many.